packages/cli/src/lib/log.ts

```diff
@@ -101,6 +103,7 @@ export function muteDevLogs({workerReload}: {workerReload?: boolean} = {}) {
   injectLogReplacer('log');
   injectLogReplacer('error');
   injectLogReplacer('warn', warningDebouncer);
+  injectLogReplacer('debug', false);
```

Comment:
This just essentially makes all `console.debug()` messages not show up?

Reply:
It adds a hook to `console.debug`. Then, if it matches any of our replacers, it modifies the message or suppresses it